tests/test_characteristic.py

```diff
@@ -172,14 +172,14 @@ def test_to_HAP_numberic(self):
     def test_to_HAP_string(self):
         char = get_char(PROPERTIES.copy())
         char.properties['Format'] = 'string'
-        char.value = 'aaa'
+        char.set_value('aaa')
         with patch.object(char, 'broker') as mock_broker:
             hap_repr = char.to_HAP()
         self.assertEqual(hap_repr['format'], 'string')
         self.assertNotIn('maxLen', hap_repr)
 
-        char.value = 'aaaaaaaaaabbbbbbbbbbccccccccccddddddddddeeeeeeeeee' \
-            'ffffffffffgggggggggg'
+        char.set_value('aaaaaaaaaabbbbbbbbbbccccccccccddddddddddeeeeeeeeee'
+                       'ffffffffffgggggggggg')
```

Comment:
Personally I would prefer `char._value = ` instead. That would make it a bit clearer what is being tested and you don't have to be that careful what is being mocked when.